src/net/bus.c

```diff
@@ -30,8 +30,12 @@ struct Bus {
   REAL* v_mag;        /**< @brief Voltage magnitude (p.u.) */
   REAL* v_ang;        /**< @brief Voltage angle (radians) */
   REAL* v_set;        /**< @brief Voltage magnitude set point (p.u.) */
-  REAL v_max;         /**< @brief Maximum voltage magnitude (p.u.) */
-  REAL v_min;         /**< @brief Minimum voltage magnitude (p.u.) */
+  REAL v_max_reg;     /**< @brief Maximum regulated voltage magnitude (p.u.) */
+  REAL v_min_reg;     /**< @brief Minimum regulated voltage magnitude (p.u.) */
+  REAL v_max_norm;    /**< @brief Normal maximum voltage magnitude violation limit (p.u.) */
+  REAL v_min_norm;    /**< @brief Normal minimum voltage magnitude violation limit (p.u.) */
+  REAL v_max_emer;    /**< @brief Emergency maximum voltage magnitude violation limit (p.u.) */
+  REAL v_min_emer;    /**< @brief Emergency minimum voltage magnitude violation limit (p.u.) */
```

Comment:
Please change docstrings to be consistent. How about
"regulation/normal/emergency maximum/minimum voltage magnitude"